featuretools/computational_backends/pandas_backend.py

```diff
@@ -518,17 +532,20 @@ def wrap(df):
             args = [df[v] for v in variable_ids]
 
             if f.primitive.uses_calc_time:
-                d[f.get_name()] = func(*args, time=time_last)
+                values = func(*args, time=time_last)
             else:
-                d[f.get_name()] = func(*args)
+                values = func(*args)
 
+            if f.primitive.number_output_features > 1:
+                names = f.get_feature_names()
+                assert len(names) == len(values)
+                d.update(dict(zip(names, values)))
```

Comment:
any reason use use DataFrame.update here instead of the loop approach from `_calculate_transform_features`? 

```
for name, value in zip(names, values):
    frame[name] = strip_values_if_series(value)
```